app/code/community/TIG/MyParcel2014/Model/Shipment.php

```diff
@@ -129,13 +129,19 @@ class TIG_MyParcel2014_Model_Shipment extends Mage_Core_Model_Abstract
     /** @var TIG_MyParcel2014_Helper_Data $helper */
     public $helper;
 
+    /**
+     * @var TIG_MyParcel2014_Helper_AddressValidation
+     */
+    private $helper_address_validation;
+
     /**
      * Initialize the shipment
      */
     public function _construct()
     {
         $this->_init('tig_myparcel/shipment');
         $this->helper = Mage::helper('tig_myparcel');
+        $this->helper_address_validation = new TIG_MyParcel2014_Helper_AddressValidation;
```

Comment:
```suggestion
        $this->helper_address_validation = new TIG_MyParcel2014_Helper_AddressValidation();
```
Moet met (). https://www.php-fig.org/psr/psr-12/#4-classes-properties-and-methods

Reply:
Done